proof-of-concept/link-wasm-object-files/main.c

```diff
@@ -0,0 +1,11 @@
+extern int return42(int a);
+
+int test(int a)
+{
+	return return42(5);
+}
+
+int main()
+{
+	return test(0);
+}
```

Comment:
* `extern` is the [default storage class](https://en.cppreference.com/w/c/language/storage_duration#:~:text=extern%20for%20all%20functions) for functions, so it is not necessary
* Use slightly more useful function as an example:


```suggestion
int add(int a, int b);

int test()
{
	return add(2, 3);
}
```

Reply:
I can see it is unnecessary, but for illustration purposes, it maybe does not harm to keep it.